<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>830</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mesh region</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QFrame" name="cfFrame">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_10">
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>cfMesh Refinement Parameters</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QFrame" name="RefinementFrame">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="frameShape">
             <enum>QFrame::Panel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="lineWidth">
             <number>0</number>
            </property>
            <layout class="QGridLayout" name="gridLayout_4">
             <property name="topMargin">
              <number>3</number>
             </property>
             <property name="bottomMargin">
              <number>5</number>
             </property>
             <item row="2" column="0">
              <widget class="QFrame" name="BoundLayerFrame">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="frameShape">
                <enum>QFrame::Panel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Plain</enum>
               </property>
               <property name="lineWidth">
                <number>0</number>
               </property>
               <layout class="QGridLayout" name="gridLayout_7">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>3</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>5</number>
                </property>
                <item row="1" column="0">
                 <layout class="QHBoxLayout" name="horizontalLayout_2">
                  <item>
                   <widget class="QLabel" name="label_expratio">
                    <property name="text">
                     <string>Expansion ratio:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QDoubleSpinBox" name="if_expratio">
                    <property name="minimum">
                     <double>1.000000000000000</double>
                    </property>
                    <property name="maximum">
                     <double>1.200000000000000</double>
                    </property>
                    <property name="singleStep">
                     <double>0.050000000000000</double>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item row="0" column="0">
                 <layout class="QHBoxLayout" name="horizontalLayout">
                  <item>
                   <widget class="QLabel" name="label_numlayer">
                    <property name="text">
                     <string>Number of layers:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSpinBox" name="if_numlayer">
                    <property name="minimum">
                     <number>1</number>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item row="2" column="0">
                 <layout class="QHBoxLayout" name="horizontalLayout_5">
                  <item>
                   <widget class="QLabel" name="label_firstlayerheight">
                    <property name="text">
                     <string>First cell height:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="Gui::InputField" name="if_firstlayerheight">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="minimumSize">
                     <size>
                      <width>80</width>
                      <height>25</height>
                     </size>
                    </property>
                    <property name="toolTip">
                     <string extracomment="Select 0 to use default value"/>
                    </property>
                    <property name="text">
                     <string>0.0</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                    </property>
                    <property name="singleStep" stdset="0">
                     <double>1.000000000000000</double>
                    </property>
                    <property name="maximum" stdset="0">
                     <double>1000000000.000000000000000</double>
                    </property>
                    <property name="unit" stdset="0">
                     <string notr="true">mm</string>
                    </property>
                    <property name="decimals" stdset="0">
                     <number>2</number>
                    </property>
                    <property name="value" stdset="0">
                     <double>0.000000000000000</double>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QCheckBox" name="check_boundlayer">
               <property name="text">
                <string>Boundary layers</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QLabel" name="label_refinethick">
                 <property name="text">
                  <string>Refinement thickness:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="Gui::InputField" name="if_refinethick">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>80</width>
                   <height>25</height>
                  </size>
                 </property>
                 <property name="toolTip">
                  <string extracomment="Select 0 to use default value"/>
                 </property>
                 <property name="text">
                  <string>0.0</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                 <property name="singleStep" stdset="0">
                  <double>1.000000000000000</double>
                 </property>
                 <property name="maximum" stdset="0">
                  <double>1000000000.000000000000000</double>
                 </property>
                 <property name="unit" stdset="0">
                  <string notr="true">mm</string>
                 </property>
                 <property name="decimals" stdset="0">
                  <number>2</number>
                 </property>
                 <property name="value" stdset="0">
                  <double>0.000000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_rellen">
              <property name="text">
               <string>Relative element size:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="if_rellen">
              <property name="maximum">
               <double>1.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.050000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QFrame" name="snappyFrame">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_9">
          <item row="1" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Type of refinement</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_5">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>snappyHexMesh Refinement parameters</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QFrame" name="frame">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout_12">
             <item row="1" column="0">
              <widget class="QRadioButton" name="snapRadio">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="text">
                <string>Snapped to surface</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QRadioButton" name="refineRadio">
               <property name="text">
                <string>Refinement region</string>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="3" column="0">
           <layout class="QGridLayout" name="gridLayout_8">
            <item row="1" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Surface Refinement Level</string>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSpinBox" name="snappyRefineLevel">
              <property name="maximum">
               <number>10</number>
              </property>
              <property name="value">
               <number>1</number>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Edge Refinement Level</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QSpinBox" name="snappySurfaceEdgeRefinementLevel"/>
            </item>
            <item row="3" column="0">
             <widget class="QCheckBox" name="baffleCheckBox">
              <property name="text">
               <string>Baffle</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
         <zorder>label_5</zorder>
         <zorder>label_7</zorder>
         <zorder>frame</zorder>
         <zorder></zorder>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QFrame" name="boundaryFrame_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="1">
           <widget class="QLabel" name="label_4">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>References</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" colspan="2">
           <widget class="QListWidget" name="list_References"/>
          </item>
          <item row="3" column="1" colspan="2">
           <widget class="QFrame" name="FaceFrame">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="frameShape">
             <enum>QFrame::Panel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="lineWidth">
             <number>0</number>
            </property>
            <layout class="QGridLayout" name="gridLayout_6">
             <property name="topMargin">
              <number>3</number>
             </property>
             <item row="0" column="0">
              <layout class="QGridLayout" name="gridLayout">
               <item row="0" column="0">
                <widget class="QLabel" name="l_label_text_5">
                 <property name="text">
                  <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Selection&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QRadioButton" name="rb_standard">
                 <property name="text">
                  <string>Face, Ed&amp;ge, Vertex</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="0" column="2">
                <widget class="QRadioButton" name="rb_solid">
                 <property name="text">
                  <string>So&amp;lid</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="pushButton_Reference">
            <property name="text">
             <string>Add reference</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="faceList">
            <property name="text">
             <string>Select from list</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
      <zorder>cfFrame</zorder>
      <zorder>cfFrame</zorder>
      <zorder>cfFrame</zorder>
      <zorder>snappyFrame</zorder>
      <zorder>boundaryFrame_2</zorder>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_11">
         <item row="0" column="2" colspan="2">
          <widget class="QComboBox" name="shapeComboBox"/>
         </item>
         <item row="1" column="0" rowspan="3" colspan="4">
          <widget class="QListWidget" name="faceListWidget">
           <property name="selectionMode">
            <enum>QAbstractItemView::ExtendedSelection</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Select Object</string>
           </property>
          </widget>
         </item>
         <item row="4" column="3">
          <widget class="QPushButton" name="addFaceListFace">
           <property name="text">
            <string>Add Face</string>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QPushButton" name="closeListOfFaces">
           <property name="text">
            <string>Back</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Gui::InputField</class>
   <extends>QLineEdit</extends>
   <header>Gui/InputField.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
